src/main/java/cceclipseplugin/editor/listeners/DirectoryListener.java

```diff
@@ -77,7 +77,7 @@ protected void handleFile(IResourceDelta delta) {
 		PluginManager pm = PluginManager.getInstance();
 		MetadataManager mm = pm.getMetadataManager();
 		FileMetadata fileMeta = mm.getFileMetadata(f.getFullPath().toString());
-		String path = f.getProjectRelativePath().toString();
+		String workspaceRelativePath = f.getProjectRelativePath().toString();
```

Comment:
getProjectRelativePath() doesn't return the workspace-relative path, getFullPath() does